ESP8266Interface.h

```diff
@@ -18,6 +18,7 @@
 #define ESP8266_INTERFACE_H
 
 #include "mbed.h"
+#include "mbed_debug.h"
```

Comment:
Hm ok, in that case can you actually move it into the cpp file for now?

It should probably be in `mbed.h` if you want to make a pr against mbed-os